bin/pre-commit-hook.js

```diff
@@ -27,6 +27,7 @@ function parseGitDiffToPathArray( command ) {
 const dirtyFiles = new Set( parseGitDiffToPathArray( 'git diff --name-only --diff-filter=ACM' ) );
 const files = parseGitDiffToPathArray( 'git diff --cached --name-only --diff-filter=ACM' );
 const phpFiles = gitFiles.filter( name => name.endsWith( '.php' ) );
+const phpcsFiles = phpFiles.filter( name => name.startsWith( '_inc/lib/debugger/' ) );
```

Comment:
I keep thinking about the best way to approach this, and I can't find the perfect solution. I wonder if we should go with a blacklist instead, and set it to exclude everything in most of the root folders of Jetpack for now. We could then slowly remove some of the paths from the blacklist as we work through them?

Reply:
I made a quick change to make an approved list: https://github.com/Automattic/jetpack/pull/11379/commits/e7d7eb6514def966c6e765c41b9a93ec6fd917e1

We can eventually make it more and more inclusive (e.g. `_inc/lib/debugger/` to `_inc/lib/` until we have full coverage?